networking/configuring-routing.adoc

```diff
@@ -0,0 +1,23 @@
+// Assembly filename: configuring-routing.adoc
+// Explains Router configuration.
+[id='configuring-routing’]
+= Configuring Routing
+include::modules/common-attributes.adoc[]
+:context: configuring-routing
+toc::[]
+
+//Creating secured routes
+include::modules/nw-creating-secured-routes.adoc[leveloffset=+1]
+//
+//Creating route timeouts
+include::modules/nw-configuring-route-timeouts.adoc[leveloffset=+1]
+//
+//Enabling HTTP Strict Transport Security
+include::modules/nw-enabling-hsts.adoc[leveloffset=+1]
+//
+//Troubleshooting Throughput Issues
+include::modules/nw-throughput-troubleshoot.adoc[leveloffset=+1]
+//
+//Troubleshooting Throughput Issues
```

Comment:
I know it's not visible, but this isn't the right heading for the following files.